internal/pkg/runtime/engine/singularity/process_linux.go

```diff
@@ -41,6 +41,22 @@ import (
 	"github.com/hpcng/singularity/pkg/sylog"
 	"github.com/hpcng/singularity/pkg/util/rlimit"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
+<<<<<<< HEAD
```

Comment:
Here's another merge error, and this one even causes compilation failures.  Please at least try to compile singularity after any change, before submitting a pull request.